packages/auth/src/api/index.ts

```diff
@@ -148,14 +148,19 @@ export async function _performApiRequest<T, V>(
       headers[HttpHeader.X_FIREBASE_LOCALE] = auth.languageCode;
     }
 
+    const fetchArgs: RequestInit = {
+      method,
+      headers,
+      ...body
+    };
+
+    if (!isCloudflareWorker()) {
```

Comment:
can we add a comment here explaining this? Maybe we should link this github issue here https://github.com/cloudflare/next-on-pages/issues/487 and mention "Security-conscious server-side frameworks tend to have built in mitigations for referrer problems"